indra/newview/llworldmapview.cpp

```diff
@@ -517,26 +517,37 @@ void LLWorldMapView::draw()
         // Draw the region name in the lower left corner
         if (mMapScale >= DRAW_TEXT_THRESHOLD)
         {
-            std::string mesg;
+            static LLCachedControl<bool> print_coords(gSavedSettings, "MapShowGridCoords");
+
+            auto print = [&](std::string text, F32 x, F32 y)
+                {
+                    LLFontGL::getFontSansSerifSmallBold()->renderUTF8(text, 0,
```

Comment:
getFontSansSerifSmallBold() is a bit expensive, please consider saving font out of draw().